docs/troubleshooting.md

```diff
@@ -6,10 +6,10 @@ The output of `kubectl describe` is helpful to see why a pod is stuck in Pending
 
 ### Virtual Kubelet Logs
 
-The a good place to look for more answers is the output of the virtual-kubelet provider pod.
+The a good place to look for more answers is the output of the kip pod.
 
 ```bash
-./kubectl -nkube-system logs virtual-kubelet -f
+./kubectl -nkube-system logs kip kip -f
```

Comment:
Did you mean "kip -c kip"?

Reply:
Now that we have 2 containers in the pod, the second argument is required.  That said, logs (unlike exec) doesn't need the -c for the second argument.  I can add it in for clarity though.